src/Actions/index.ts

```diff
@@ -135,10 +135,11 @@ export function getUrlsList({ token }: { token: string }) {
 type InputParams = {
   token: string;
   longUrl: string;
-  props: any;
 };
-export function Input_Url_Api({ token, longUrl, props }: InputParams) {
-  console.log(props);
+export function Input_Url_Api(
```

Comment:
let's change the function name to match the pattern of `signup`, `editUrl` ....etc actions